source/includes/_cluster.md

```diff
@@ -0,0 +1,419 @@
+      "host_ip": "172.20.0.2",
+      "node_name": "manager",
+      "encryption_key": "<REDACTED>"
+  })
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```shell
+$ curl -X PUT -u user:password -H "Content-Type: application/json" -d '{"host_ip": "172.20.0.2", "node_name": "manager", "encryption_key": "<REDACTED>"}' http://<manager-ip>/api/v3/cluster
+```
+
+
+```html
+obsolete
+```
+> Response Example
+
+```json
+{
+    "initialized": false
+}
+```
+
+`PUT "{manager-ip}/api/v3/cluster"`
+
+Starts the cluster mechanisms on the current manager: either joins an existing
+cluster, if the `join_addrs` parameter is provided, otherwise bootstraps a new
+cluster.
+Only admin users can execute this operation.
+
+### Request Body
+
+Property | Type | Description
+-------- | ---- | -----------
+host_ip | string | The externally-reachable IP of this node.
+node_name | string | A unique name for this node to be used internally within the cluster.
+encryption_key | string | A base64-encoded 16-byte encryption key, identical for all the nodes in the cluster.
+join_addrs | list | IPs of the nodes to join to; if not provided, a new cluster will be created.
+
+
+### Response
+
+A `ClusterState` resource.
+
+
+## Patch Cluster State
+
+> Request Example
+```python
+client.cluster.update(
+    config_key='config_value'
+)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/",
+  "method": "PATCH",
+  "headers": headers,
+  "contentType": "application/json"
+  "data": JSON.stringify({
+      "config_key": "config_value"
+  })
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```shell
+$ curl -X PATCH -u user:password -H "Content-Type: application/json" -d '{"config_key": "config_value"}' http://<manager-ip>/api/v3/cluster
+```
+
+```html
+obsolete
+```
+
+> Response Example
+```json
+{
+    "initialized": true,
+    "error": null,
+    "encryption_key": "<REDACTED>"
+}
+```
+
+`PATCH "{manager-ip}/api/v3/cluster"`
+
+Update the cluster configuration. The request body is a mapping containing
+arbitrary settings, which can be used by either the core cluster mechanisms,
+or user-specific extensions, if any.
+Only admin users can execute this operation.
+
+### Response
+
+A `ClusterState` resource.
+
+
+## The ClusterNode resource
+
+The ClusterNode resource represents the state of a node in the cluster
+
+### Attributes:
+
+Attribute | Type | Description
+--------- | ---- | -----------
+`master`| boolean | Whether this node is the current cluster master.
+`name` | string | The name of this node.
+`host_ip` | string | The externally-reachable IP of this node.
+`online` | boolean | Whether this node is currently online.
+`initialized` | boolean | Whether the node has joined the cluster successfully.
```

Comment:
Whether the node has ~joined the~ **been successfully joined to the** cluster ~successfully~.